ibm/data_source_ibm_pi_image.go

```diff
@@ -47,6 +47,10 @@ func dataSourceIBMPIImage() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"storage_type": {
```

Comment:
Add this storage_type  attribute to the docs of both resource and datasources pi_image